contrib/test_jenkins.sh

```diff
@@ -738,19 +732,16 @@ run_ucx_perftest() {
 	# client/server mode, to reduce testing time
 	if [ "X$have_cuda" == "Xyes" ] && [ $with_mpi -ne 1 ]
 	then
-		tls_list="all "
 		gdr_options="n "
 		if (lsmod | grep -q "nv_peer_mem")
 		then
 			echo "GPUDirectRDMA module (nv_peer_mem) is present.."
-			tls_list+="rc,cuda_copy "
 			gdr_options+="y "
 		fi
 
 		if  [ "X$have_gdrcopy" == "Xyes" ] && (lsmod | grep -q "gdrdrv")
 		then
 			echo "GDRCopy module (gdrdrv) is present..."
-			tls_list+="rc,cuda_copy,gdr_copy "
```

Comment:
why need checking GDRCopy? it doesn't change variants

Reply:
it affects "export UCX_IB_GPU_DIRECT_RDMA=$gdr"

Reply:
right, removing it